services/blockchain-indexer/shared/indexer/transactionProcessor/dpos/unlockToken.js

```diff
@@ -21,8 +21,14 @@ const commandName = 'unlockToken';
 const applyTransaction = async (blockHeader, tx, dbTrx) => {
 };
 
+// eslint-disable-next-line no-unused-vars
+const revertTransaction = async (blockHeader, tx, dbTrx) => {
```

Comment:
this logic is meant to handle the block deletion?

Reply:
This logic is added to handle transactions on every delete block event. Basically, if any module specific operations needs to perform on delete block, we can add here.